client/src/app/site/pages/meetings/pages/motions/services/export/motion-xlsx-export.service/motion-xlsx-export.service.ts

```diff
@@ -187,7 +187,7 @@ export class MotionXlsxExportService {
                 data.push(
                     ...commentIds.map(commentId => {
                         if (commentId === PERSONAL_NOTE_ID) {
-                            return motion && motion.getPersonalNote()! && motion.getPersonalNote()!.note;
+                            return (motion && motion.getPersonalNote()! && motion.getPersonalNote()!.note) ?? ``;
```

Comment:
Why so complicated? 

```suggestion
                            return motion?.getPersonalNote()?.note ?? ``;
```

Reply:
I tried to fix the bug and was not in code refactoring mode.

Reply:
But you are right, I will look at this.